resources/js/pack-it-forms.js

```diff
@@ -189,14 +189,14 @@ function parse_form_data_text(text) {
         if (line.match(/^\s*$/)) {
             return;  // Ignore empty lines
         }
-        if (line.match(/^!PACF!/)) {
-            return;  // Ignore PACF line as we don't need anything from it.
-        }
         if (line.match(/^!OUTPOST! /)) {
             // Grab outpost data fields and store for substitution
             outpost_envelope = outpost_envelope_to_object(line);
             return;
         }
+        if (line.match(/^!.*!/)) {
+            return;  // Ignore line as we don't need anything from it.
+        }
```

Comment:
I realize the code wasn't doing this before, but given we're expanding the set of options that will be matched by the regular expression, what do you think about adding `&& linenum == 0` since I believe this has to be the very first line?

Reply:
The cases I've seen so far are the first non-empty line and the last non-empty line. I'm inclined to make the parser relaxed. I appeal to the [robustness principle](https://en.wikipedia.org/wiki/Robustness_principle). And it appears Jim Oberhofer likes to use ! to introduce lines that aren't fields, so the subsequent code that expects a field is likely off track.

Reply:
Fair enough. This should work fine now and hopeflly we'll have warning if there are planned changes that will affect it.